Volha_Ivanova/Module_1_Java/TestNG/src/com/epam/cdp/volha_ivanova/task1/test/DivTestNegative.java

```diff
@@ -0,0 +1,16 @@
+package com.epam.cdp.volha_ivanova.task1.test;
+
+import org.testng.annotations.Test;
+
+public class DivTestNegative extends BaseCalculatorTest {
```

Comment:
usually there is one test class for  one class. And name like `ClassNameTests`
I understand that you waned to split a lot of cases. But you should know about best practice